githubkit/typing.py

```diff
@@ -27,19 +27,19 @@
 PrimitiveData: TypeAlias = Optional[Union[str, int, float, bool]]
 QueryParamTypes: TypeAlias = Union[
     httpx.QueryParams,
-    dict[str, Union[PrimitiveData, list[PrimitiveData]]],
-    list[tuple[str, PrimitiveData]],
+    Mapping[str, Union[PrimitiveData, Sequence[PrimitiveData]]],
+    Sequence[tuple[str, PrimitiveData]],
     tuple[tuple[str, PrimitiveData], ...],
     str,
     bytes,
 ]
 
 HeaderTypes: TypeAlias = Union[
     httpx.Headers,
-    dict[str, str],
-    dict[bytes, bytes],
-    list[tuple[str, str]],
-    list[tuple[bytes, bytes]],
+    Mapping[str, str],
+    Mapping[bytes, bytes],
+    Sequence[tuple[str, str]],
+    Sequence[tuple[bytes, bytes]],
 ]
 
 CookieTypes: TypeAlias = Union[httpx.Cookies, dict[str, str], list[tuple[str, str]]]
```

Comment:
Note: The httpx methods that accept CookieTypes define these as `dict` and `list` too, so these can't be broadened.